station/submitobs.py

```diff
@@ -88,41 +146,45 @@ def submit_observation(data: SubmitRequestData):
         "Authorization": header_value
     }
 
-    files = {
-        "file": (
-            filename,
-            file_obj,
-            "image/png",
-            {}
-        )
-    }
+    # Check if notes are a valid JSON
+    logging.debug(f"Meta-data = {data.config}")
 
-    logging.info("Submitting observation, url=%s file=%s" % (url, filename))
+    logging.info(f"Submitting observation, url={url}, file(s)={data.image_path}")
```

Comment:
It looks verbose. Use the debug level.

Reply:
This is very useful piece of information and usually there's only one file. Are you sure about debug?